pkg/volume/cinder/attacher_test.go

```diff
@@ -175,6 +181,46 @@ func TestAttachDetach(t *testing.T) {
 			expectedError: diskPathError,
 		},
 
+		// Successful VolumesAreAttached call, attached
+		{
+			name:             "VolumesAreAttached_Positive",
+			instanceID:       instanceID,
+			disksAreAttached: disksAreAttachedCall{[]string{diskName}, instanceID, map[string]bool{diskName: true}, nil},
+			test: func(testcase *testcase) (string, error) {
+				attacher := newAttacher(testcase)
+				attachments, err := attacher.VolumesAreAttached([]*volume.Spec{spec}, nodeName)
+				return serializeAttachments(attachments), err
```

Comment:
> serializeAttachments

Does not look like the best thing to do, open for suggestions on what is idiomatic in Go